test/Microsoft.OpenApi.Tests/PublicApi/PublicApi.approved.txt

```diff
@@ -1379,6 +1379,7 @@ namespace Microsoft.OpenApi.Writers
         public bool InlineLocalReferences { get; set; }
         [System.Obsolete("Use InlineLocalReference and InlineExternalReference settings instead")]
         public Microsoft.OpenApi.Writers.ReferenceInlineSetting ReferenceInline { get; set; }
+        public bool Terse { get; set; }
```

Comment:
Why is this moved up into OpenApiWriterSettings?  Terse is only something that makes sense for JSON.  You can't remove whitespace from YAML.

Reply:
Thank you for the correction, resolving...
I think the confusion was because while switching the format to JSON before writing out, the OpenApiJsonWriter consumes the OpenApiWriter settings to do the serialization hence I thought it easier to add a Terse parameter in those settings. I've updated the PR with a better approach factoring in your comment.